### PR TITLE
MRG, ENH: Add extract_chpi_locs_kit

### PR DESCRIPTION
**Pull request by larsoner**
Will enable head movement processing for KIT. Locally this works already, but `compute_head_pos` fails because `chpi` needs to be refactored.

- [x] ~~Ideally extract n_coils (maybe~~ it's always 5 from the instance itself if mrk/hsp are provided
- [x] ~~Ideally extract the channel name to look for measurement times rather than hard-code or~~ provide idx param
- [x] Modify cHPI code to handle KIT (no `hpi_subsystem` or `hpi_meas`)
- [x] Add testing data
- [x] Add tests

Next PR:

- Add example of head movement compensation for KIT
- Add example of head movement compensation for CTF

### CODE REVIEW

File: doc/changes/names.inc [hunk 1]
```diff
@@ -266,6 +266,10 @@
 
 .. _Robert Luke: https://github.com/rob-luke
 
+.. _Robert Seymour: https://neurofractal.github.io
+
+.. _Matt Sanderson: https://www.linkedin.com.au/in/matt-sanderson-0a3667180
```
Comment (larsoner) [inline doc/changes/names.inc:271]: @monkeyman192 I want to credit your original work on the code here (shared with me by Paul Sowman), is this a good URL to use for you?

**Comment by larsoner** [comment]
@agramfort this is ready for review/merge from my end. I plan to make examples using CTF and/or KIT data in a follow-up PR rather than here (they require folks at other sites making new recordings so the ETA is a bit up in the air).

File: doc/changes/names.inc (hunk 1 above)
Comment (larsoner) [reply doc/changes/names.inc:271]: (If it's not good for you, feel free to make a quick PR to my branch or "suggest" in GitHub and I'm happy to incorporate. This also has the advantage of adding your name/email in the commit history which is nice.)

Comment (monkeyman192) [reply doc/changes/names.inc:271]: Woo! Glad this code is finally going to good use!
Umm, maybe using my github page would be more consistent here. I'll make a suggestion, thanks!

Comment (larsoner) [reply doc/changes/names.inc:271]: If you made a suggestion it did not show up, maybe you forgot to click "comment" or "Finish Review" or something?

**Review by agramfort — APPROVED** [review]
merge if happy @larsoner

**Comment by larsoner** [comment]
@monkeyman192 feel free to make a GitHub suggestion or PR into my branch so that you can get some code credit. I'll probably merge Monday or Tuesday

**Review by monkeyman192 — CHANGES_REQUESTED** [review]

File: doc/changes/names.inc (hunk 1 above)
Comment (monkeyman192) [reply doc/changes/names.inc:271]: Sorry! I thought I had put it through. It appears I did in fact forget to press 'finish review'!

Comment (larsoner) [reply doc/changes/names.inc]: Yes, I have done this for several days and also completely lost my suggestions before so you're not alone!

**Pull request merged**